ndcube/utils/wcs.py

```diff
@@ -32,8 +32,20 @@ def inv(self):
 # types and International Virtual Observatory Alliance vocabulary.
 # See http://www.ivoa.net/documents/REC/UCD/UCDlist-20070402.html
 wcs_to_ivoa = {
+    "DSUN": "custom:pos.distance.sunToObserver",
+    "HGLT": "custom:pos.heliographic.stonyhurst.lon",
+    "CRLT": "custom:pos.heliographic.carrington.lat",
+    "HGLN": "custom:pos.heliographic.stonyhurst.lat",
+    "CRLN": "custom:pos.heliographic.carrington.lon",
+    "HECR": "custom:pos.distance.fromSunCenter",
+    "HECR": "custom:pos.distance.fromSunSurface",
+    "SOLX": "custom.pos.distance.heliocentric.west",
+    "SOLY": "custom.pos.distance.heliocentric.north",
+    "SOLZ": "custom.pos.distance.heliocentric.sunToObserver",
     "HPLT": "custom:pos.helioprojective.lat",
     "HPLN": "custom:pos.helioprojective.lon",
+    "HPRZ": "custom:pos.distance.helioprojective.sunToObserver",
```

Comment:
Does above comments on ```"SOLZ"``` mean ```"HPRZ"``` should be ```"custom:pos.helioprojective.z"```?  Perhaps ```"custom:pos.helioprojective.alt"```?

Reply:
I think "z" personally.